src/viz-sentence-lengths/VizSentenceLengths.js

```diff
@@ -15,7 +15,7 @@ export default function VizSentenceLengths({
   dimension,
   locationId,
 }) {
-  if (!dimension || !locationId) return null;
```

Comment:
Why'd we remove this check? Does this still do what we want for all users?

Reply:
it does, I tested it on the sentencing page ... it's a bit of a cheat honestly, I was pleasantly surprised that it actually worked. 

`locationId` was initially an "optional" prop because it takes a couple of render cycles for it to populate when the page components are first mounted (because it gets fed through the state of the dropdown). To support the prison page (which has no locations) it needed to be fully optional, as in the component could render without it. And it turns out that it can, the only change this caused is that there is an animated transition when the chart initially populates. (There are other ways to stop that from happening if we don't like it, but in most cases you are not likely to see it anyway because the chart won't be onscreen initially, you would have to scroll down very quickly when the page first loads.)